modules/darwin/src/main/ts/ephox/darwin/api/InputHandlers.ts

```diff
@@ -53,6 +53,11 @@ const keyboard = (win: Window, container: SugarElement, isRoot: (e: SugarElement
     const shiftKey = realEvent.shiftKey === true;
 
     const handler = CellSelection.retrieve(container, annotations.selectedSelector).fold(() => {
+      // Make sure any possible lingering annotations are cleared
+      if (SelectionKeys.isNavigation(keycode) && shiftKey === false) {
```

Comment:
This is a guaranteed boolean right?

```suggestion
      if (SelectionKeys.isNavigation(keycode) && !shiftKey) {
```